Furball.Game/FurballTestGame.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using Furball.Engine;
 using Furball.Engine.Engine.Localization;
 using Furball.Game.Screens;
+using Silk.NET.Windowing;
 
 namespace Furball.Game {
     public class FurballTestGame : FurballGame {
-        public FurballTestGame() : base(new ScreenSelector()) {
-            this.Window.AllowUserResizing = true;
+        public FurballTestGame() : base(new ScreenSelector(), WindowOptions.Default) {
+            //this.Window.AllowUserResizing = true;
```

Comment:
Perhaps the second parameter should be optional

Reply:
given the constructor no longer needs it anyway, removed